### PR TITLE
[IMP] Add qweb report for settlements

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

File: sale_commission/report/settlements_report.xml [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+            <field name="res_model">sale.commission.settlement</field>
+            <field name="view_mode">tree,form</field>
+        </record>
+
+        <menuitem id="menu_all_settle_commissions"
+                  parent="menu_sale_commissions_conf"
+                  action="action_all_settle_commissions"/>
+
+    </data>
+</openerp>
```
Comment (mileo) [inline sale_commission/report/settlements_report.xml]: @Hendrixcosta No newline at the end of file

File: sale_commission/i18n/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+#: view:sale.commission.make.invoice:sale_commission.sale_commission_make_invoice_form
+#: field:sale.commission.make.invoice,settlements:0
+#: view:sale.commission.settlement:sale_commission.view_settlement_tree
+msgid "Settlements"
+msgstr "Settlements"
+
+#. module: sale_commission
+#: field:sale.commission.settlement,state:0
+msgid "State"
+msgstr "Estado"
```
Comment (mileo) [inline sale_commission/i18n/pt_BR.po]: Can you change for "situação"

**Comment by pedrobaeza** [comment]
There's another similar PR for this: https://github.com/OCA/commission/pull/91

**Comment by pedrobaeza** [comment]
Is this going to be finished?

**Comment by pedrobaeza** [comment]
No answer, so I close.

**Pull request closed without merging**